package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "tsoa",
+  "name": "sl-tsoa",
```

Comment:
Let's keep this just 'tsoa' since once it gets merged in, it won't be in your fork